packages/password/src/accounts-password.ts

```diff
@@ -88,6 +88,12 @@ export interface AccountsPasswordOptions {
    * Will automatically send a verification email after signup.
    * Default to false.
    */
+
+  removeAllPasswordResetTokensAfterPasswordChanged?: boolean;
+  /**
```

Comment:
looks like this comment is not for the good option ;)

Reply:
oops :)